packages/firestore/test/unit/model/object_value_builder.test.ts

```diff
@@ -17,78 +17,75 @@
 
 import { expect } from 'chai';
 
-import {
-  ObjectValue,
-  ObjectValueBuilder
-} from '../../../src/model/object_value';
+import { ObjectValue } from '../../../src/model/object_value';
 import { field, wrap, wrapObject } from '../../util/helpers';
 
-describe('ObjectValueBuilder', () => {
-  it('supports empty builders', () => {
-    const builder = new ObjectValueBuilder();
-    const object = builder.build();
-    expect(object.isEqual(ObjectValue.empty())).to.be.true;
+describe('MutableObjectValue', () => {
```

Comment:
I would like to rename this file to `mutable_object_value.test.ts` but GitHub then sees it as a new file, so I will do this after the first review.